homeassistant/components/shelly/config_flow.py

```diff
@@ -50,7 +51,12 @@
     mac_address_from_name,
 )
 
-HOST_SCHEMA: Final = vol.Schema({vol.Required(CONF_HOST): str})
+HOST_SCHEMA: Final = vol.Schema(
```

Comment:
```suggestion
CONFIG_SCHEMA: Final = vol.Schema(
```
It is no longer just host

Reply:
Fixed